Sources/SuperwallKit/Misc/Extensions/SKProduct+Helpers.swift

```diff
@@ -110,105 +110,121 @@ extension SKProduct {
     return ""
   }
 
-  var periodWeeks: String {
+  var periodWeeks: Int {
     guard let subscriptionPeriod = subscriptionPeriod else {
-      return ""
+      return 0
     }
 
     let numberOfUnits = subscriptionPeriod.numberOfUnits
 
     if subscriptionPeriod.unit == .day {
-      return "\(Int((1 * numberOfUnits) / 7))"
+      return (1 * numberOfUnits) / 7
     }
 
     if subscriptionPeriod.unit == .week {
-      return "\(Int(numberOfUnits))"
+      return numberOfUnits
     }
 
     if subscriptionPeriod.unit == .month {
-      return "\(Int(4 * numberOfUnits))"
+      return 4 * numberOfUnits
     }
 
     if subscriptionPeriod.unit == .year {
-      return "\(Int(52 * numberOfUnits))"
+      return 52 * numberOfUnits
     }
 
-    return "0"
+    return 0
+  }
+
+  var periodWeeksString: String {
+    return "\(periodWeeks)"
   }
 
-  var periodMonths: String {
+  var periodMonths: Int {
     guard let subscriptionPeriod = subscriptionPeriod else {
-      return ""
+      return 0
     }
     let numberOfUnits = subscriptionPeriod.numberOfUnits
 
     if subscriptionPeriod.unit == .day {
-      return "\(Int((1 * numberOfUnits) / 30))"
+      return (1 * numberOfUnits) / 30
     }
 
     if subscriptionPeriod.unit == .week {
-      return "\(Int(numberOfUnits / 4))"
+      return numberOfUnits / 4
     }
 
     if subscriptionPeriod.unit == .month {
-      return "\(Int(numberOfUnits))"
+      return numberOfUnits
```

Comment:
Can you include something along the lines of 
```
if subscriptionPeriod.unit == .month {
  if subscriptionPeriod.numberOfUnits == 3 {
    return "quarter"
  } else {
    return "month"
  }
}
```
to account for "per quarter" / "quarterly" options in the paywall editor. Currently customers who have quarterly but use the "period" var are seeing "monthly" in 2.x. We may also want to consider supporting 2 months ("bi-monthly") and 6 months ("bi-yearly") because those are other supported Subscription Duration periods.

Reply:
Okay added in quarter like you said. Let's leave the bimonthly/biyearly because they can mean both twice a month/year or once every two months/years! I think that'll get confusing. Also, removed this SKProduct+Helpers file because that's not actually used anymore now that we have StoreProduct.